server/src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -1197,7 +1198,7 @@ public AsyncJobStatus executeJob(JobMessage message) throws JobException {
         // rather than failing directly. This would allow use of aliases and explicit job
         // classes.
         if (jobClass == null) {
-            String errmsg = String.format("No registered job class for job: %s", status.getJobKey());
+            String errmsg = String.format("No registered job class for job_key=%s", status.getJobKey());
```

Comment:
I'm guessing the intent here is to make this particular line searchable as part of the same log query? If so, changing the message isn't the way to go here. This block can safely live in the try block below immediately following the ```setupJobRuntimeEnvironment``` call (with proper whitespace).

I also noticed a bug unrelated to your code in the log message below regarding job instantiation. We can fix that here as well:

```
        try {
            this.setupJobRuntimeEnvironment(status);

            // Maybe in this case it'd be better to attempt to use the job key as the job class
            // rather than failing directly. This would allow use of aliases and explicit job
            // classes.
            Class<? extends AsyncJob> jobClass = getJobClass(status.getJobKey());
            if (jobClass == null) {
                String errmsg = String.format("No registered job class for job: %s", status.getJobKey());

                this.updateJobStatus(status, JobState.FAILED, errmsg);

                log.error(errmsg);
                throw new JobInitializationException(errmsg, true);
            }

            EventSink eventSink = injector.getInstance(EventSink.class);
            AsyncJob job = injector.getInstance(jobClass);

            if (job == null) {
                String errmsg = String.format("Unable to instantiate job class \"%s\" for job: %s",
                    jobClass.getName(), status.getJobKey());

                log.error(errmsg);
                throw new JobInitializationException(errmsg);
            }
```

Reply:
Done.